data/json/itemgroups/Locations_MapExtras/mansion.json

```
@@ -79,6 +79,7 @@
       { "group": "jewelry_front", "prob": 40 },
       { "group": "costume_jewelry_vampire", "prob": 5 },
       { "group": "costume_jewelry_dark", "prob": 5 },
+      { "group": "mansion_exotic_weapons", "prob": 75 },
```

Not at this rate, this is 14% of all mansion safe spawns which is quite high.
```suggestion
      { "group": "mansion_exotic_weapons", "prob": 5 },
```